File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Body/ApiUserRegisterReturn.kt
package SampleProject.Protocol.Body

import SampleProject.Protocol.Context.*
open class ApiUserRegisterReturn: AbstractReturn() {
    var UserCurrencyTypeEnumId: Int = (CurrencyTypes.EUR.rawValue).toInt()

    var Token: ApiTokenPostReturn? = null

    var ErrorCodes: ArrayList<String>? = null
}

File: SampleSetup/Output/TypeScript/SampleProject/Protocol/Body/ApiUserRegisterReturn.ts
import { AbstractReturn } from './AbstractReturn';
import { ApiTokenPostReturn } from './ApiTokenPostReturn';
import { CurrencyTypes } from '../Context/CurrencyTypes';
export class ApiUserRegisterReturn extends AbstractReturn {
    UserCurrencyTypeEnumId: number = CurrencyTypes.EUR as number;

    Token: ApiTokenPostReturn | null = null;

    ErrorCodes: Array<string> | null = null;
}
